server.js

```diff
@@ -0,0 +1,291 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const qs = require('querystring');
+
+let authenticationObject = {
+  something: 'imadethisup',
+};
+
+let existingFiles = ['css', '404.html', 'index.html', '.keep'];
+
+let elementNameReceived = '';
+let elementSymbolReceived = '';
+let elementAtomicNumberReceived = 0;
+let elementDescriptionReceived = '';
+let fileName = '';
+
+const server = http.createServer((req, res) => {
+  let reqURL = '/index.html';
+
+  if (req.url !== '/') {
+    if (fs.existsSync(`./public${req.url}`)) {
```

Comment:
You should not be using existsSync. I know I forgot to mention it but try to re-write this section to not use it.